### PR TITLE
Shrinking default root partition size + updating osmosis hash

### PR DESCRIPTION
**Pull request by susyo**

### CODE REVIEW

**Comment by noam-stratoscale** [comment]
@eliran-stratoscale  @shimshon-stratoscale 
pls review, this is done for ticket 
https://stratoscale.atlassian.net/browse/STOR-663
Approved By Shlomp.
after merging, we need to have a clean build for this

File: inaugurator/partitiontable.py [hunk 1]
```diff
@@ -9,8 +9,9 @@ class PartitionTable:
     _DEFAULT_SIZES_GB = dict(
         smallSwap=1,
         bigSwap=8,
-        minimumRoot=14,
-        createRoot=30)
+        minimumRoot=12,
+        maximumRoot=18,
```
Comment (rom-stratoscale) [inline inaugurator/partitiontable.py]: why do u set maximum to 18? Why cannot it be 100?

File: inaugurator/partitiontable.py [hunk 2]
```diff
@@ -9,8 +9,9 @@ class PartitionTable:
     _DEFAULT_SIZES_GB = dict(
         smallSwap=1,
         bigSwap=8,
```
Comment (rom-stratoscale) [inline inaugurator/partitiontable.py]: what is the meaning of swap? How do we use it (It's not part of the CR).
@eliran-stratoscale? @shimshon-stratoscale ?

File: inaugurator/partitiontable.py [hunk 3]
```diff
@@ -9,8 +9,9 @@ class PartitionTable:
...
-        minimumRoot=14,
-        createRoot=30)
+        minimumRoot=12,
+        maximumRoot=18,
+        createRoot=15)
```
Comment (rom-stratoscale) [inline inaugurator/partitiontable.py]: If you touching this part - why not to parametrize it?
Anyhow, not sure I would go for it before we understand how it affects the inaugurations - how much storage space the system consumes once you move everything to data partition?
Another option is to create another cache partition for the inauguration cache.

**Pull request closed without merging**